OpenSim/Simulation/Model/AbstractTool.cpp

```diff
@@ -553,22 +553,31 @@ printResults(const string &aBaseName,const string &aDir,double aDT,
 }
 
 
-bool AbstractTool::createExternalLoads( const string& aExternalLoadsFileName, Model& aModel, const Storage *loadKinematics)
+bool AbstractTool::createExternalLoads( const string& aExternalLoadsFileName,
+                                        Model& aModel, const Storage *loadKinematics)
 {
     if(aExternalLoadsFileName==""||aExternalLoadsFileName=="Unassigned") {
         cout<<"No external loads will be applied (external loads file not specified)."<<endl;
         return false;
     }
 
+    Model copyModel = aModel;
+    // speedup realize position calculations by removing all force elements
+    // including muscles whose path calculations are most intensive
+    copyModel.updForceSet().clearAndDestroy();
+    copyModel.updControllerSet().clearAndDestroy();
+
     // This is required so that the references to other files inside ExternalLoads file are interpreted 
     // as relative paths
     std::string savedCwd = IO::getCwd();
     IO::chDir(IO::getParentDirectory(aExternalLoadsFileName));
     // Create external forces
+    ExternalLoads* externalLoads = nullptr;
     try {
-        _externalLoads = ExternalLoads(aModel, aExternalLoadsFileName);
+        externalLoads = new ExternalLoads(aExternalLoadsFileName, true);
+        copyModel.addModelComponent(externalLoads);
     }
```

Comment:
This will put ExternalForce(s) in the miscModelComponents bag, does that have side effects downstream?

Reply:
I would not call it a side-effect, but one might assume that `ExternalForces` live in  the model's `ForceSet`, which was what IAA was doing. The fix is simple and we need to move away from assuming the organization of the Model and rely on/use  `getComponenList<C>`to obtain all components of certain type.

Reply:
@aseth1 does that explain or relate to the RRA adjusted model problems you mention in https://github.com/opensim-org/opensim-gui/issues/1008